devtools-ci-index.yaml

```diff
@@ -4221,8 +4221,12 @@
             git_repo: fabric8-analytics-release-monitor
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-analytics
+            deployment_units: 'f8a-release-monitor'
```

Comment:
This should be `fabric8-analytics-release-monitor`: https://github.com/openshiftio/saas-analytics/blob/master/bay-services/release-monitor.yaml#L4